scripts/clang-build-analyzer

```diff
@@ -34,8 +34,9 @@ git diff | git -C "${WORKING_DIR}" apply
 # Configure VAST to build with time-traces enabled
 pushd "${WORKING_DIR}"
 on_exit=('popd' "${on_exit[@]}")
-./configure "${@}" --build-dir="${WORKING_DIR}/build" \
-  --extra-flags="-ftime-trace"
+cmake "${@}" -B "${WORKING_DIR}/build" \
+  -DCMAKE_CXX_COMPILER="clang++" \
```

Comment:
This may break for some setups. Please do not hardcode the compiler in scripts.

Reply:
But `-ftime-trace` is not understood by g++ and results in an error.

Reply:
Hence why we have a `VAST_ENABLE_TIME_TRACE` option.